tests/check-http-method.spec.js

```diff
@@ -0,0 +1,20 @@
+const HTTP_METHODS = require('../lib/http-methods');
+const checkHttpMethod = require('../lib/check-http-method');
+
+describe('checkHttpMethod function ->', () => {
+    HTTP_METHODS.forEach(method => {
+        it(`should checked as true for "${method}" method`, done => {
```

Comment:
should treat HTTP method "${method}" as a correct method

Reply:
+